tests/webpack.config.js

```diff
@@ -0,0 +1,61 @@
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const path = require('path');
+
+module.exports = env => ({
+  mode: 'development',
+  entry: path.resolve(__dirname, './webpack-tests.js'),
+  resolve: {
+    alias: {
+      'fsProvider': path.resolve(__dirname, '../shims/providers/default'),
```

Comment:
Forgive my webpack ignorance here: can we not use Filer in this setup the way you're outlining in the README?  It would be great if we had test code that did what the README suggests, so we know if it breaks.

Reply:
Do you mean aliasing fs, path and Buffer? We can definitely do that but obviously we'd need to refactor the shim tests to require fs, path and Buffer instead of the shims so those tests would have to be separated from the rest of the tests. I think that makes sense though as they're webpack specific.

Reply:
Right, that's what I was thinking. Given that there are a few ways to do this, having adequate test coverage of all of them is my goal.

Reply:
Hey @humphd, I'm still working on this PR but found an issue with the way we're shimming the path module. In short, the alias messes up the import of the path module in filer (which I'd previously overlooked) and as such the path module ends up missing a lot of the methods (all the ones not explicitly replaced by filer). It should be possible to write a webpack plugin which resolves the path module correctly. Are you happy for me to go ahead with this and to make a separate PR for it?

Reply:
@bcheidemann I think that sounds like a good idea.  Thanks for pushing on this code a bit, it's great that testing is already discovering ways we can make it better.

Reply:
No problem! I'm happy to do it and it's teaching me a lot :D I just wish I had more time to work on it at the moment!